lib/swoosh/adapter.ex

```diff
@@ -22,6 +24,21 @@ defmodule Swoosh.Adapter do
         expected #{inspect key} to be set, got: #{inspect config}
         """
       end
+
+      defp code_to_reason(400), do: :bad_request
+      defp code_to_reason(401), do: :unauthorized
```

Comment:
@stevedomin wouldn't be better to remove all the functions out of the macro and do some `import` or something inside?

Reply:
Hi @yordis, sorry for the slow response, been travelling.

Definitely, I still need to make a lot of improvement to that PR.